Web/Pages/CCGMember/Compare/ObjectiveLevel.cshtml

```diff
@@ -0,0 +1,70 @@
+﻿@page
+@using Microsoft.AspNetCore.Http
+@model Web.Pages.CCGMember.Compare.ObjectiveLevelModel
+@inject IHttpContextAccessor contextAccessor
+@inject IViewLocalizer Localizer
+@{
+    var isEnglish = System.Threading.Thread.CurrentThread.CurrentCulture.TwoLetterISOLanguageName == "en";
+    var id = Model.id;
+    var jobGroupPositions = Model.JobGroupPositions;
+    var levels = new List<string>();
+    var JobGroupName = "";
+    if (isEnglish)
+    {
+        JobGroupName = Model.JobGroup.NameEng;
+    }
+    else
+    {
+        JobGroupName = Model.JobGroup.NameFre;
+    }
+    foreach (var jp in jobGroupPositions)
+    {
+        levels.Add(jp.LevelValue);
+    }
+    var levelsd = levels.Distinct();
```

Comment:
Variables need to have descriptive names. Instead of levelsd it can be called distinctLevels